packages/modules/src/firebase/checkout.ts

```diff
@@ -0,0 +1,49 @@
+import type { Request, Response } from 'firebase-functions';
+import { ValidateFunction } from 'ajv';
```

Comment:
```suggestion
import type { ValidateFunction } from 'ajv';
```

Melhor importar só o `type`, aí não tem esse import no output em JS